samples/StaticHtmlComposites/README.md

```diff
@@ -0,0 +1,46 @@
+# Basic sample for using CallComposite and ChatComposite
+
+This sample is the bare minimum to embed Calling or Chat in your web app without needing to set up React. It consists of the following files:
```

Comment:
This sample uses react but mentions here 'without needing to setup react'. Is this because our current quickstarts all use the CRA?

Reply:
Given this still uses the server I'm worried this gives of the wrong impression of the 'bare minimum'

Reply:
Rephrased the intro to make it more clear that it still uses React but hides it and that this is intended for those who don't build React apps (but still want to use composites).

Also added yet another (third?) callout that the server portion is missing/fake.